applications/CableNetApplication/tests/test_empirical_spring.py

```diff
@@ -0,0 +1,308 @@
+from __future__ import print_function, absolute_import, division
+import KratosMultiphysics
+
+
+import KratosMultiphysics.CableNetApplication as CableNetApplication
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+from KratosMultiphysics.CableNetApplication import empirical_spring_element_process
```

Comment:
this is a test, hence you have to check if numpy polywhatever is available before importing it